docs/articles/documentation/reference/test-api/testcontroller/expect/notok.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: docs
+title: t.expect.notOk Method
+permalink: /documentation/reference/test-api/testcontroller/expect/notok.html
+---
+# t.expect.notOk Method
+
+Asserts that `actual` is false.
+
+```text
+await t.expect( actual ).notOk( message, options );
+```
+
+Parameter              | Type                                              | Description
+---------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------
+`actual`             | Any type | A value tested in the assertion. The assertion passes if the `actual` value is falsy. See [`actual` parameter value](#actual-parameter-value).
```

Comment:
```suggestion
`actual`             | Any type | A value tested in the assertion. The assertion passes if the `actual` value is false. See [`actual` parameter value](#actual-parameter-value).
```

Reply:
@dirk-pieterse `falsy` is more relevant in this context. More info on this [here](https://developer.mozilla.org/en-US/docs/Glossary/Falsy).